src/components/MediaCollection/createTrackMediaSession.js

```diff
@@ -0,0 +1,43 @@
+export default ({
+  config,
+  logger,
+  mediaEventManager,
+  mediaSessionCacheManager,
+  legacy = false
+}) => {
+  return options => {
+    if (!config.mediaCollection) {
+      logger.warn("Media Collection is not configured.");
```

Comment:
Throw an error or return a Promise.reject and let the executeCommand logic log an error.